bower.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "webfontloader",
+  "main": "webfontloader.js",
+  "version": "1.6.21",
+  "homepage": "https://github.com/typekit/webfontloader",
+  "authors": [
```

Comment:
Do we really need to duplicate this stuff? It's already in the package.json file.